### PR TITLE
cmd, core, params: add support for the Holesky testnet

### PR DESCRIPTION
**Pull request by karalabe**

### CODE REVIEW

**Review by parithosh — APPROVED** [review]

File: core/genesis.go [hunk 1]
```diff
@@ -576,6 +576,19 @@ func DefaultSepoliaGenesisBlock() *Genesis {
 	}
 }
 
+// DefaultHoleskyGenesisBlock returns the Holesky network genesis block.
+func DefaultHoleskyGenesisBlock() *Genesis {
+	return &Genesis{
+		Config:     params.HoleskyChainConfig,
+		Nonce:      0x1234,
+		ExtraData:  hexutil.MustDecode("0x686f77206d7563682069732074686520666973683f"),
```
Comment (holiman) [inline core/genesis.go:584]: `how much is the fish?`

Comment (parithosh) [reply core/genesis.go:584]: shhhh.....

File: params/config.go [hunk 2]
```diff
@@ -58,6 +59,31 @@ var (
...
+		MuirGlacierBlock:              nil,
+		BerlinBlock:                   big.NewInt(0),
+		LondonBlock:                   big.NewInt(0),
+		ArrowGlacierBlock:             nil,
+		GrayGlacierBlock:              nil,
+		TerminalTotalDifficulty:       big.NewInt(0),
+		TerminalTotalDifficultyPassed: true,
+		MergeNetsplitBlock:            nil,
+		ShanghaiTime:                  newUint64(1694790240),
+		CancunTime:                    newUint64(2000000000),
```
Comment (holiman) [inline params/config.go:84]: From https://github.com/eth-clients/holesky: 

> Cancun: (TBD)

**Review by holiman — APPROVED** [review]
Looks good to me

File: params/config.go (hunk 2 above)
Comment (barnabasbusa) [reply params/config.go:84]: Cancun 2033 🥇

File: core/genesis.go (hunk 1 above)
Comment (q9f) [reply core/genesis.go:584]: we only added this to check if you review this carefully

**Comment by Deacon77** [comment]
I’m learning! I still make many mistakes! Hole in the sky, hole in my soul!On Sep 4, 2023, at 4:45 AM, Afri ***@***.***> wrote:﻿
@q9f commented on this pull request.



In core/genesis.go:
> @@ -576,6 +576,19 @@ func DefaultSepoliaGenesisBlock() *Genesis {
 	}
 }
 
+// DefaultHoleskyGenesisBlock returns the Holesky network genesis block.
+func DefaultHoleskyGenesisBlock() *Genesis {
+	return &Genesis{
+		Config:     params.HoleskyChainConfig,
+		Nonce:      0x1234,
+		ExtraData:  hexutil.MustDecode("0x686f77206d7563682069732074686520666973683f"),

we only added this to check if you review this carefully

**Pull request merged**